src/ucp/wireup/select.c

```diff
@@ -188,7 +188,7 @@ ucp_wireup_select_transport(ucp_ep_h ep, const ucp_address_entry_t *address_list
     }
 
     if (!addr_index_map) {
-         snprintf(p, endp - p, "not supported by peer  ");
+         snprintf(p, endp - p, "%s  ", ucs_status_string(UCS_ERR_UNSUPPORTED));
```

Comment:
this is not same error meaning, if we get here it means the **remote peer** does not support something, and not the local worker